utils/loadthread.cpp

```diff
@@ -166,8 +164,8 @@ void LoadThread::load2D(nix::DataArray array, nix::NDSize start, nix::NDSize ext
             }
             start[xDimIndex] = offset + i * chunksize;
 
-            array.getData(array.dataType(),chunkdata.data(),extent, start);
-
+            //array.getData(array.dataType(),chunkdata.data(),extent, start);
+            array.getData(chunkdata, extent, start);
             loadedData.insert(loadedData.end(), chunkdata.begin(), chunkdata.end());
```

Comment:
This is what i'm expecting to have caused the segfaults at least without actually testing it.

Reply:
yes, that was my first impression as well, but the data that was loaded is 1D, so this function was never called